test/__snapshots__/index.spec.ts.snap

```diff
@@ -704,6 +704,120 @@ Array [
 ]
 `;
 
+exports[`jest rules jest/prefer-expect-assertions should fail on an invalid fixture 1`] = `
+Array [
+  Object {
+    "column": 1,
+    "endColumn": 5,
+    "endLine": 1,
+    "fix": Object {
+      "range": Array [
+        0,
+        4,
+      ],
+      "text": "it",
+    },
+    "line": 1,
+    "message": "Prefer using 'it' instead of 'test'",
```

Comment:
This doesn't seem necessary for these tests.

Reply:
no, removed with: https://github.com/ridedott/eslint-config/pull/367/commits/1aa15a86da56ffde9db436ec67f09cda4e3b61ea